### ISSUE TITLE
Add functions to get anchors from factomd anchors API

### ISSUE THREAD

**Issue by ilzheev** [FactomProject/factom#125]
In factomd:v6.4.3 new API call appeared:
https://docs.factom.com/api#anchors

```
Retrieve information about the directory block anchors that have been confirmed on Bitcoin and Ethereum.

Parameter options: - "height" - the directory block height (integer) to request anchors for - "hash" - the object’s hash (hex string) to request anchors for (e.g. entry hash, entry block keymr, factoid block keymr, admin block lookup hash, entry credit block header hash, or directory block keymr)
```

I suggest to add 2 functions into factom lib:
`GetAnchors(hash string)` and `GetAnchorsByHeight(height int64)`

**Comment by ilzheev** [comment]
@Emyrk @stackdump @WhoSoup bump ^^^

**Comment by WhoSoup** [comment]
> @Emyrk @stackdump @WhoSoup bump ^^^

i'm swamped with other work atm so if anyone else would like to take a stab at it please do. feel free to ping me again if it hasn't gotten done by end of next week

**Comment by PaulBernier** [comment]
Has been merged, will be available in release 0.4
